### PR TITLE
HRSPLT-142 : Copy changes from student preferred name for user delete fu...

### PR DESCRIPTION
**Pull request by timlevett**
...nction call.

### CODE REVIEW

**Comment by apetro** [comment]
Feasible to backlog somewhere re-architecting to remove duplication between different kinds of Preferred Name portlets?

**Comment by timlevett** [comment]
~~sure, i'll do that.~~ In theory, when we move to the new portlet for "personal information" we will have done that.

File: hrs-portlets-bnsemail-impl/src/main/java/edu/wisc/prefname/dao/PreferredNameDaoImpl.java [hunk 1]
```diff
@@ -34,6 +35,11 @@ public void setJdbcTemplate(@Qualifier("prefname") NamedParameterJdbcOperations
     public void setUpdatePreferredEmail(UpdatePreferredNameProcedure updatePrefNameProc) {
         this.updatePreferredName = updatePrefNameProc;
     }
+	
+	@Autowired
+    public void setDeletePreferredEmail(DeletePreferredNameUserFunction deletePreferredName) {
```
Comment (apetro) [inline hrs-portlets-bnsemail-impl/src/main/java/edu/wisc/prefname/dao/PreferredNameDaoImpl.java]: Mention of `email` in the property name seems off.  `setDeletePreferredNameUserFunction` instead?

Comment (timlevett) [inline hrs-portlets-bnsemail-impl/src/main/java/edu/wisc/prefname/dao/PreferredNameDaoImpl.java]: fixed

**Comment by vertein** [comment]
+1

**Pull request merged**